scripts/v2/make-mi-fic.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+function create_role_assignment() {
+  az role assignment create --assignee "${USER_ASSIGNED_OBJECT_ID}" \
+      --role "Owner" \
+      --subscription "${AZURE_SUBSCRIPTION_ID}"
+}
+
+function retry_create_role_assignment() {
+  until create_role_assignment; do
+    sleep 5
+  done
+
+}
+
+print_usage() {
+  echo "Usage: make-mi-fic-kind-wi-storage.sh -g <RESOURCE_GROUP> -i <ISSUER> -d <DIR>"
```

Comment:
What do `mi` and `wi` mean? Also, doesn't match filename.

Reply:
managed identity/workload identity